allzpark/control.py

```diff
@@ -1165,8 +1164,6 @@ def _try_resolve_context(req, pkg_name, mode):
             for app_request in apps:
 
                 app_package = _try_finding_latest_app(app_request)
-                if package_filter.excludes(app_package):
-                    continue
```

Comment:
Just leaving notes:

While I was resolving merge conflict, I found #129 actually also *accidentally* fixed the issue that this PR is aiming to. In #129, instead of using `_rez_api.find_latest`, it has changed to use contorller's `find` method which already filtering packages with filter.

Still, removing this line here is good, it's redundant anyway.